Source/player.cpp

```diff
@@ -164,15 +164,44 @@ struct DirectionSettings {
 	void (*walkModeHandler)(int, const DirectionSettings &);
 };
 
+// clang-format off

```

Comment:
Based on [other areas of the codebase](https://github.com/diasurgical/devilutionX/blob/32d39c65246b17fbf6b89d4a914beb04f02ee7e8/Source/drlg_l1.cpp#L141-L156), these clang-format comments should be inside the array initializers. I assume this avoids the possibility of slipping formatting errors into the left-hand side of the variable definition.

For example...

```c++
/** Specifies the frame of each animation for which an action is triggered, for each player class. */
const int PlrGFXAnimLens[enum_size<HeroClass>::value][11] = {
	// clang-format off
	{ 10, 16,  8,  2, 20, 20,  6, 20,  8,  9, 14 },
	{  8, 18,  8,  4, 20, 16,  7, 20,  8, 10, 12 },
	{  8, 16,  8,  6, 20, 12,  8, 20,  8, 12,  8 },
	{  8, 16,  8,  3, 20, 18,  6, 20,  8, 12, 13 },
	{  8, 18,  8,  4, 20, 16,  7, 20,  8, 10, 12 },
	{ 10, 16,  8,  2, 20, 20,  6, 20,  8,  9, 14 },
	// clang-format on
};
```